app/javascript/components/AdminRouter.jsx

```diff
@@ -27,14 +27,14 @@ export default class AdminRouter extends React.Component {
       <BrowserRouter>
         <div>
           <Route exact path="/admin" render={(props) => {
-            return <AdminPage auth={this.auth} {...props} /> 
+            return(<AdminPage auth={this.auth} {...props} />);
           }}/>
           <Route path="/admin/organization_overview" render={(props) => {
-            return <OrganizationOverview auth={this.auth} {...props} /> 
+            return(<OrganizationOverview auth={this.auth} {...props} />);
           }}/>
           <Route path="/admin/callback" render={(props) => {
             this.handleAuthentication(props);
-            return <Callback {...props} /> 
+            return(<Callback {...props} />);
```

Comment:
In de toekomst kunnen we eslint nog toevoegen aan CI

Reply:
Ja dat is wel hip. Ik maak wel een ticket aan.